wordpress-mobile/WordPress-iOS
# Stats Today widget: update to use new Stats

Fixes #12756

This updates the Today widget to use new Stats data.
- For fetching data, it now calls the `StatsServiceRemoteV2:getInsight` method.
- For caching data:
  - The `TodayWidgetStats` struct stores the data in `TodayData.plist`.
  - The widget loads saved data from that plist, and updates it after fetching.
  - When Insights are fetched within the app, Today data is saved to the plist for the widget to access.

---
To test:
- Go to Stats > Insights.
- Note the data for _Today_ Visitors and Views.
- Select 'Today' on the nav bar.
- Select 'Use this site'.
- Exit the app.
- If not yet added, add the WordPress widget to your device's Today View. (See below for steps)
- Verify the data displayed matches what was in Insights _Today_.

<img width="400" alt="stats_no_data" src="https://user-images.githubusercontent.com/1816888/67438393-96872800-f5b0-11e9-9eac-058c516c7b95.png">

<img width="400" alt="widget_no_data" src="https://user-images.githubusercontent.com/1816888/67438392-96872800-f5b0-11e9-856a-a7a03b4bc299.png">

<img width="400" alt="stats_data" src="https://user-images.githubusercontent.com/1816888/67438477-d0582e80-f5b0-11e9-8229-a8dc74e576d1.png">

<img width="400" alt="widget_data" src="https://user-images.githubusercontent.com/1816888/67438476-d0582e80-f5b0-11e9-8906-1d12a923cd83.png">

---
How to add a widget:
- On your device, scroll to the left-most screen (the Today View).
- Tap edit.
- Scroll down to WordPress, and select the green button.
- Select 'Done'.

---
Update release notes:
